### ISSUE TITLE
Bump Scala.js version in sbt-scalajs-crossproject to 0.6.19

### ISSUE THREAD

**Issue by mdedetrich** [scala-native/sbt-crossproject#50]
Receiving the following error

```
[warn] 	* org.scala-js:sbt-scalajs:0.6.19 is selected over 0.6.16
[warn] 	    +- default:scalajson-build:0.1-SNAPSHOT (scalaVersion=2.10, sbtVersion=0.13) (depends on 0.6.19)
[warn] 	    +- org.scala-native:sbt-scalajs-crossproject:0.2.0 (scalaVersion=2.10, sbtVersion=0.13) (depends on 0.6.16)
```

**Comment by sjrd** [comment]
That's not an error. It's a warning, and the warning is wrong. Scala.js versions are binary compatible across minor releases. Ignore it.

If it bothers you, file a bug report to sbt so that they stop those erroneous warnings.

**Comment by mdedetrich** [comment]
Right, but does this mean that my project is being compiled with 0.6.19 or 0.6.16 (because the former has bug fixes and also compiler performance improvements iirc)

**Comment by sjrd** [comment]
The message says pretty explicitly that 0.6.19 is used. In general, the bigger version number is always used.

**Comment by mdedetrich** [comment]
@sjrd For some reason I read the error message the wrong way in my head, probably shouldn't comment on tickets when sleep deprived :p

**Comment by densh** [comment]
Fixed in 0.2.2
